Uno.Gallery/Uno.Gallery.UWP/App.xaml.cs

```diff
@@ -99,7 +99,32 @@ private void OnLaunchedOrActivated()
 
 			// Ensure the current window is active
 			window.Activate();
-		}		
+		}
+
+		public static void TryNavigateToLaunchSample(string title, string design)
+		{
+			const string UndefinedValue = "undefined";
+
+			if (!HasValue(title))
+			{
+				return;
+			}
+
+			var sample = GetSamples().FirstOrDefault(s => s.ViewType.Name.ToLowerInvariant() == title.ToLowerInvariant());
```

Comment:
This logic is very similar to the one in `BranchService.cs`, there might be a method to extract.